tests/pipeline/test_filter.py

```diff
@@ -395,6 +395,36 @@ def test_isfinite(self):
         )
         check_arrays(results['isfinite'], isfinite(data))
 
+    def test_smoothing_filter(self):
+        from zipline.pipeline.filters import SmoothingFilter
+
+        data = full(self.default_shape, True, dtype=bool)
+        # one column all false
```

Comment:
This comment isn't true, you have an array with the first two diagonal values False.